emperor/support_files/js/sceneplotview3d.js

```diff
@@ -81,72 +86,83 @@ define([
      */
     this.visibleDimensions = _.clone(this.decViews.scatter.visibleDimensions);
 
-    /**
-     * Ranges for all the decompositions in this view (there's a min and a max
-     * property).
-     * @type {Object}
-     */
-    this.dimensionRanges = {'max': [], 'min': []};
-    this._unionRanges();
-
     // used to name the axis lines/labels in the scene
     this._axisPrefix = 'emperor-axis-line-';
     this._axisLabelPrefix = 'emperor-axis-label-';
 
-    // Set up the camera
-    var max = _.max(decViews.scatter.decomp.dimensionRanges.max);
-    var frontFrust = _.min([max * 0.001, 1]);
-    var backFrust = _.max([max * 100, 100]);
-
-    /**
-     * Camera used to display the scene.
-     * @type {THREE.PerspectiveCamera}
-     */
-    // these are placeholders that are later updated in updateCameraAspectRatio
-    this.camera = new THREE.OrthographicCamera(-50, 50, 50, -50);
-    this.camera.position.set(0, 0, max * 5);
-    this.camera.zoom = 0.7;
-
     //need to initialize the scene
     this.scene = new THREE.Scene();
-    this.scene.add(this.camera);
     this.scene.background = new THREE.Color(this.backgroundColor);
 
     /**
-     * Object used to light the scene, by default is set to a light and
+     * Camera used to display the scatter scene.
+     * @type {THREE.OrthographicCamera}
+     */
+    this.scatterCam = this.buildCamera('scatter');
+
+    /**
+     * Object used to light the scene in scatter mode,
+     * by default is set to a light and
      * transparent color (0x99999999).
      * @type {THREE.DirectionalLight}
      */
     this.light = new THREE.DirectionalLight(0x999999, 2);
     this.light.position.set(1, 1, 1).normalize();
-    this.camera.add(this.light);
+    this.scatterCam.add(this.light);
+
+    /**
+     * Camera used to display the parallel plot scene.
+     * @type {THREE.OrthographicCamera}
+     */
+    this.parallelCam = this.buildCamera('parallel-plot');
```

Comment:
Would you mind explaining the rationale for having two cameras instead of having one that changes some of its attributes depending on the current UI state?